src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -218,51 +218,55 @@ def load_command_table(self, _):
 
     # region DNS
     with self.command_group('network dns', network_dns_reference_sdk, resource_type=ResourceType.MGMT_NETWORK_DNS) as g:
-        g.command('list-references', 'get_by_target_resources')
+        g.custom_command('list-references', 'get_by_target_resources')
 
     with self.command_group('network dns zone', network_dns_zone_sdk) as g:
-        g.command('delete', 'begin_delete', confirmation=True)
-        g.show_command('show', 'get', table_transformer=transform_dns_zone_table_output)
+        g.custom_command('delete', 'delete_dns_zone', confirmation=True)
+        g.custom_show_command('show', 'show_dns_zone', table_transformer=transform_dns_zone_table_output)
```

Comment:
It's not a good practice to use custom function to wrap it, recommended to use inherent way. E.g., https://github.com/Azure/azure-cli/blob/b4f88827759d59eb20c590264a919aa185cfc65c/src/azure-cli/azure/cli/command_modules/network/commands.py#L144. Similar in other places.

Reply:
I used custom wrapper for Create, Update and Delete APIs because they have some deprecated arguments which is not supported by AAZ.